toolkit/tools/imagegen/configuration/partitionflag.go

```diff
@@ -20,6 +20,8 @@ const (
 	PartitionFlagGrub PartitionFlag = "grub"
 	// PartitionFlagBiosGrub indicates this is a bios grub boot partition
 	PartitionFlagBiosGrub PartitionFlag = "bios_grub"
+	// PartitionFlagBiosGrubLegacy indicates this is a bios grub boot partition. Needed to preserve legacy config behavior.
+	PartitionFlagBiosGrubLegacy PartitionFlag = "bios-grub"
```

Comment:
>PartitionFlagBiosGrubLegacy PartitionFlag = "bios-grub" [](start = 1, length = 55)

I presume this is to fix a breaking change...and it's too late to fix the breaking change (since it already escaped into the world?)

Reply:
Yes. At some point, we may want to consider updating our calamares package to use the `bios_grub` flag but today the calamares packages are using our older `bios-grub` and those calamares packages are in the wild. @MateuszMalisz for awareness.